packages/gatsby-plugin-sharp/README.md

```diff
@@ -147,6 +152,26 @@ following:
 Convert the source image to one of the following available options: `NO_CHANGE`,
 `JPG`, `PNG`, `WEBP`.
 
+#### toFormatBase64
+
+base64 image uses the image format from the source , or the value of `toFormat`. This setting allows a different image format instead, available options are: `JPG`, `PNG`, `WEBP`.
```

Comment:
in code they are lowercased and there is no case insensitive comparison is used

Reply:
It uses uppercase like the docs above it for `toFormat`. It calls `healOptions()` to lowercase just like `toFormat` does.